src/app/containers/StoryPromo/LinkContents/__snapshots__/index.test.jsx.snap

```diff
@@ -19,7 +19,6 @@ exports[`Story Promo Link Contents should render with visually hidden text for m
   <span
     class="c0"
   >
-    AUDIO
```

Comment:
Do we expect this span to only contain a comma `,`?

Reply:
The mock wasn't updated to include the translation for listen. It's been updated now